config/initializers/devise.rb

```diff
@@ -296,4 +297,63 @@
   # When set to false, does not sign a user in automatically after their password is
   # changed. Defaults to true, so a user is signed in automatically after changing a password.
   # config.sign_in_after_change_password = true
+
```

Comment:
Remove whitespace. Also, can the commented boilerplate Devise configs be removed? Or moved to another file for use when authentication is configurable (e.g. LDAP vs SAML)?

Reply:
I like having all the configuration comments in the file. It is a really easy way to remind yourself what can be set and what the default is.